x-pack/plugins/profiling_data_access/common/stack_traces.ts

```diff
@@ -24,7 +24,6 @@ export interface ProfilingStatusResponse {
   };
   resources: {
     created: boolean;
-    pre_8_9_1_data: boolean;
```

Comment:
Was this expected to be removed? Just curious since I recall the PR adding it was recent.

Reply:
Good catch. This should not be removed.